controllers/v1/admin.js

```diff
@@ -167,6 +168,7 @@ module.exports = class Admin {
 				let deleteResource = await adminHelper.deletedResourceDetails(
 					req.params._id,
 					req.query.type,
+					req.query.isAPrivateProgram ? req.query.isAPrivateProgram : 'false',
```

Comment:
@MallanagoudaB why you are treating boolean as string ? 'false' -> false. if needed UTILS.convertStringToBoolean conversion do it here